book/listings/css/6/style.css

```diff
@@ -0,0 +1,6 @@
+button.button_1:hover {
+  color: red;
+  background-color: yellow;
+  background-image: none;
+  transition: 1s;
```

Comment:
FWIW it's a bad idea to add transition specifically on hover. This means you don't get the same transition when transitioning _from_ hover to another state. If you change transition, you want to do that in general, not just for hover.

Also, this transition affects things such as focus-ring, you probably want to style specifically background/color instead.